app/models/active_guard.rb

```diff
@@ -1,6 +1,6 @@
 class ActiveGuard < Clearance::SignInGuard
   def call
-    if current_user.deleted?
+    if current_user.present? && current_user.deleted?
```

Comment:
This still doesn't work, since `current_user.present?` evaluating to true puts us into the `next_guard` branch, which will continue along with sign in.